evm_loader/program/src/instruction.rs

```diff
@@ -353,6 +359,7 @@ impl<'a> EvmInstruction<'a> {
                 let step_count = step_count.try_into().ok().map(u64::from_le_bytes).ok_or(InvalidInstructionData)?;
                 EvmInstruction::ExecuteTrxFromAccountDataIterativeV02 {collateral_pool_index, step_count}
             },
+            23 => EvmInstruction::UpdateValidsTable,
```

Comment:
Does the indexer know about the new instruction?

Reply:
This instruction doesn't influence to execution model in the indexer.
But to be totally correct indexer should process it.
I created the issue neonlabsorg/proxy-model.py#301